source/MaterialXTest/MaterialXRenderGlsl/RenderGlsl.cpp

```diff
@@ -551,6 +560,34 @@ bool GlslShaderRenderTester::runRenderer(const std::string& shaderName,
     return true;
 }
 
+void  GlslShaderRenderTester::runBake(mx::DocumentPtr doc, const mx::FileSearchPath& imageSearchPath, const mx::FilePath& outputFileName,
```

Comment:
GLSL only bake for now. But interface set up for bakers later on. Assuem TextureBaker would need to be refactored a bit.